lib/rust/ensogl/component/grid-view/src/visible_area.rs

```diff
@@ -38,18 +38,39 @@ pub fn visible_columns(
     col_count: usize,
     column_widths: &ColumnWidths,
 ) -> Range<Col> {
-    let column_left = |idx: usize| idx as f32 * entry_size.x + column_widths.pos_offset(idx);
+    let entry_width = entry_size.x;
+    let column_left = |idx: usize| idx as f32 * entry_width + column_widths.pos_offset(idx);
     let column_right = |idx: usize| column_left(idx + 1);
```

Comment:
Technically, we can call `column_widths.pos_offset(idx);` with the `idx` being equal to col_cound. Does our structure allow that? If yes, add the information about the fact to the `pos_offset` method docs.

Reply:
Yes, we can pass `idx == col_count`. Extended documentation of that method.